index.html

```diff
@@ -10,16 +10,12 @@
   <body>
     <span class="cercle1"></span>
     <span class="cercle2"></span>
-
-    <section>
-      <nav>
-        <h1>JK</h1>
-        <div class="toggle">
-          <span></span>
-        </div>
-      </nav>
-    </section>
-
+    <nav>
+      <h1>JK</h1>
+      <div class="toggle">
```

Comment:
- [ ] Instead of using span here, You can use the `hamburger` icon here. Refer [Font Awesome](https://fontawesome.com/) website & This `hamburger` icon should be inside the `nav` tag.